### PR TITLE
chore: clean up migrated BigQuery samples and update related README.md

### PR DESCRIPTION
**Pull request by stephaniewang526**
Towards #4803

Detailed [discussion](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/4803#issuecomment-809752753)

### CODE REVIEW

File: bigquery/README.md [hunk 1]
```diff
@@ -1,43 +1,12 @@
 # Getting Started with BigQuery
 
-Google's BigQuery Service features a REST-based API that allows developers to
-create applications to run ad-hoc queries on massive datasets. These sample
-Java applications demonstrate how to access the BigQuery API.
+he samples have been moved to live alongside the Java client libraries for Cloud BigQuery.
```
Comment (lesv) [inline bigquery/README.md:3]: The ?

File: bigquery/README.md [hunk 2]
```diff
@@ -1,43 +1,12 @@
...
 
-## API Libraries
+BigQuery: 
+https://github.com/googleapis/java-bigquery/tree/master/samples/snippets/src/main/java/com/example/bigquery
```
Comment (lesv) [inline bigquery/README.md:6]: [BigQuery](https://github.com/googleapis/java-bigquery/tree/master/samples/snippets/src/main/java/com/example/bigquery)

File: bigquery/README.md [hunk 3]
```diff
@@ -1,43 +1,12 @@
...
-## Products
-- [Google BigQuery][2]
-
-## Language
-- [Java][3]
-
-[2]: https://cloud.google.com/bigquery
-[3]: https://java.com
+BigQueryStorage:
+https://github.com/googleapis/java-bigquerystorage/tree/master/samples/snippets/src/main/java/com/example/bigquerystorage
```
Comment (lesv) [inline bigquery/README.md:9]: link instead?

**Review by lesv — APPROVED** [review]
* Fix the comment
* Consider changing to `[...](https://...)` instead of raw URL's.

File: bigquery/README.md (hunk 1 above)
Comment (stephaniewang526) [reply bigquery/README.md]: yes thanks - corrected

File: bigquery/README.md (hunk 2 above)
Comment (stephaniewang526) [reply bigquery/README.md]: done

File: bigquery/README.md (hunk 3 above)
Comment (stephaniewang526) [reply bigquery/README.md]: done

**Pull request merged**